js/client.js

```diff
@@ -1,50 +1,10 @@
 import 'babel-polyfill'
-import { init, gameLoop, toggleSound } from './engine';
-import { getPreviewBoard } from './simulator';
-require('./console');
+import React from 'react';
+import ReactDOM from 'react-dom';
 
+import { SpaceShooter } from './SpaceShooter';
```

Comment:
```suggestion
import SpaceShooter from './SpaceShooter';
```

Reply:
Let's just `export default` at the component level if we can. I think this makes it more clear if component files export more than one thing.